### PR TITLE
RDCC-3657: Refactoring to implement Java 11 features

### PR DESCRIPTION
**Pull request by LukeHalsall**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-3657

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by ishratt — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java [hunk 1]
```diff
@@ -156,7 +156,7 @@ void setUp() {
         userRoles.add("pui-user-manager");
 
         newUserCreationRequest = new NewUserCreationRequest("some-name", "some-last-name",
-                "some@email.com", userRoles, false);
+                "some@email.com", userRoles,false);
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java:159]: seems the formatting is off here?

**Review by amitkumarshrivastava — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationStatusValidatorImplTest.java [hunk 2]
```diff
@@ -88,7 +88,7 @@ void test_validateAndReturnStatusList_invalid_status_check() {
     }
 
     void verifyException(String status, String message) {
-        assertThatThrownBy(() -> organisationStatusValidatorImpl.validateAndReturnStatusList(status))
+        Assertions.assertThatThrownBy(() -> organisationStatusValidatorImpl.validateAndReturnStatusList(status))
```
Comment (AdilOoze) [inline src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationStatusValidatorImplTest.java:91]: Please revert back to the static import that was being used previously

**Review by SujeethKumarAri — APPROVED** [review]

**Review by ManishParyani — DISMISSED** [review]

File: src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationStatusValidatorImplTest.java (hunk 2 above)
Comment (LukeHalsall) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/controller/request/validator/impl/OrganisationStatusValidatorImplTest.java]: Fixed.

File: src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java (hunk 1 above)
Comment (LukeHalsall) [reply src/test/java/uk/gov/hmcts/reform/professionalapi/controller/external/OrganisationExternalControllerTest.java]: Fixed.

**Review by AdilOoze — APPROVED** [review]

**Review by ManishParyani — APPROVED** [review]

**Pull request merged**